cmd/kubeadm/app/master/selfhosted.go

```diff
@@ -331,7 +323,13 @@ func getSchedulerDeployment(cfg *kubeadmapi.MasterConfiguration) ext.Deployment
 			},
 		},
 	}
-	return d
+}
+
+func prependSelfHostedFlock(name string, args []string) []string {
+	// TODO: flock has to be inside the container, and a shell must exist with this approach
+	// Since the flock binary is present at /bin/flock in a busybox image and /usr/bin/flock in a debian image, we can't hardcode the path
```

Comment:
Also, we have previously had some fairly vague clarification about a lock being required for self-hosting, now we don't have any clarification and just a pointless comment about path of a pretty standard command not being hardcode.